cli/integration/BUILD

```diff
@@ -8,6 +8,13 @@ go_library(
         "runner.go",
         "test_file.go",
     ],
+    embedsrcs = [
```

Comment:
> we put "golden files" in the temp dir
We can discuss how best to handle the golden files. I did not understand what would be the benefit of adding them to the temp config dir. Embedding them makes handling the paths a bit easier. 

@lucapette Could you help me understand the reasoning behind the quoted point a bit more? :)

Reply:
because right now it changes the git status